src/App.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
+import { PostList } from './components/PostList';
 
 import './App.scss';
 
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
+const fullPosts = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.userId),
+  comments: [...comments].filter(comment => comment.postId === post.id),
```

Comment:
[...comments]
explain why do you need it

Reply:
Misha showed it on video, add advised us do it when we don't need all props of object